Tests/ReactiveSwiftTests/ActionSpec.swift

```diff
@@ -58,6 +58,23 @@ class ActionSpec: QuickSpec {
 				action.errors.observeValues { errors.append($0) }
 				action.completed.observeValues { completedCount += 1 }
 			}
+			
+			it("should retain the state property") {
+				var property: MutableProperty<Bool>? = MutableProperty(false)
+				weak var weakProperty = property
+				
+				var action: Action<(), (), NoError>? = Action(state: property!, enabledIf: { _ in true }) { _ in
```

Comment:
This line is triggering a "variable is written to but never read" warning. Mind changing this to a `let` constant? Probably doesn't need to be optional either.